src/nwb_benchmarks/benchmarks/network_tracking_remote_slicing.py

```diff
@@ -7,9 +7,13 @@
     get_object_by_name,
     get_s3_url,
     network_activity_tracker,
+    read_hdf5_fsspec_with_cache,
     read_hdf5_nwbfile_fsspec_no_cache,
+    read_hdf5_nwbfile_fsspec_with_cache,
     read_hdf5_nwbfile_remfile,
+    read_hdf5_nwbfile_remfile_with_cache,
     read_hdf5_nwbfile_ros3,
+    read_hdf5_remfile_with_cache,
```

Comment:
We are currently not testing slicing against the raw hdf5 reads, only against the files that are opened with PyNWB so we don't need the imports for `read_hdf5_fsspec_with_cache` and `read_hdf5_remfile_with_cache` here. I'll submit a fix shortly.